.bin/set-up-globals.sh

```diff
@@ -1,12 +1,35 @@
 #!/bin/bash
-set -ex
-
+set -e
 
 TERMINUS_PLUGINS_DIR=.. terminus list -n remote
 
-set +ex
-# TODO: $TERMINUS_SITE is not set yet. We still need to set up a CI fixture for running tests against.
-echo "Test site is $TERMINUS_SITE"
 echo "Logging in with a machine token:"
 terminus auth:login -n --machine-token="$TERMINUS_TOKEN"
 terminus whoami
+terminus multidev:create "$TERMINUS_SITE".dev ci-"$BUILD_NUM"
```

Comment:
This many be a bit bike-sheddy, but does building multidev environments belong in the "set up globals" script?

Reply:
¯\_(ツ)_/¯ the script filename isn't super representative of it's current purpose. I believe it was a file that was created as part of the `terminus plugin:create` command that just got repurposed. It could be re-named to just `setup.sh`